### PR TITLE
feat: implement logistic regression

### PR DESCRIPTION
**Pull request by masdevas**

### CODE REVIEW

File: include/athena/operation/GEMMOperation.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+#ifndef ATHENA_GEMMOPERATION_H
+#define ATHENA_GEMMOPERATION_H
+
+#include <athena/core/operation/Operation.h>
+#include <athena/operation/internal/GEMMOperationInternal.h>
+#include <athena/operation/operation_export.h>
+
+namespace athena::operation {
+class ATH_OPERATION_EXPORT GEMMOperation : public core::Operation {
```
Comment (alexbatashev) [inline include/athena/operation/GEMMOperation.h:22]: I think it sounds better this way.
```suggestion
class ATH_OPERATION_EXPORT MatmulOperation : public core::Operation {
```

File: src/operation/internal/CopyOperationInternal.cpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    utils::SharedPtr<core::internal::ContextInternal> context,
+    Generator& generator,
+    std::vector<utils::Index>& operationArguments,
+    GenNode parentNode) const {
+  GenValue input = parentNode.getOperand(0);
+  GenValue out = parentNode.getResult();
+
+  generator.setInsertionPoint(parentNode);
+
+  // TODO support other data types
```
Comment (alexbatashev) [inline src/operation/internal/CopyOperationInternal.cpp:56]: At the moment, all data types are supported. The divergence occurs when you create a constant. Then you have to explicitly specify constant type that is being passed to `createConstant`.

File: tests/Testing/Temporary/LastTest.log [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Start testing: May 24 17:30 MSK
```
Comment (alexbatashev) [inline tests/Testing/Temporary/LastTest.log:1]: Remove this

File: tests/Testing/Temporary/CTestCostData.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+---
```
Comment (alexbatashev) [inline tests/Testing/Temporary/CTestCostData.txt:1]: Remove this

File: utils/dialects/include/AthenaGraph/AthenaGraphOps.td [hunk 5]
```diff
@@ -243,6 +243,26 @@ def AthenaGraph_AddOp : AthenaGraph_Op<"add", [ComputationalOpInterface]> {
   }];
 }
 
+def AthenaGraph_CopyOp : AthenaGraph_Op<"copy", [ComputationalOpInterface]> {
+let summary = "Element-wise addition operation";
```
Comment (alexbatashev) [inline utils/dialects/include/AthenaGraph/AthenaGraphOps.td:247]:
```suggestion
let summary = "Element-wise copy operation";
```

File: utils/dialects/include/AthenaGraph/AthenaGraphOps.td [hunk 6]
```diff
@@ -243,6 +243,26 @@ def AthenaGraph_AddOp : AthenaGraph_Op<"add", [ComputationalOpInterface]> {
...
+let description = [{ TBD }];
```
Comment (alexbatashev) [inline utils/dialects/include/AthenaGraph/AthenaGraphOps.td:248]: Some indentation would be nice

File: src/operation/internal/CopyOperationInternal.cpp (hunk 2 above)
Comment (masdevas) [reply src/operation/internal/CopyOperationInternal.cpp:56]: Ok. Here is just extra comment in the code. I will remove it.

File: CMakeModules/AthenaRuntime.cmake [hunk 7]
```diff
@@ -20,7 +20,7 @@ function(athena_disable_exceptions target_name)
     if (WIN32)
         set(NEW_COMPILE_OPTIONS "/EHsc-")
     else()
-        set(NEW_COMPILE_OPTIONS "-fno-exceptions")
```
Comment (alexbatashev) [inline CMakeModules/AthenaRuntime.cmake:23]: How and why?!

**Pull request merged**